charts/consul/templates/mesh-gateway-deployment.yaml

```diff
@@ -225,13 +225,14 @@ spec:
               mountPath: /consul/tls/ca
               readOnly: true
             {{- end }}
```

Comment:
```yaml
          {{- if .Values.meshGateway.initServiceInitContainer.resources }}
          resources: {{ toYaml .Values.meshGateway.initServiceInitContainer.resources | nindent 12 }}
          {{- end }}
```

I think we can just do something like ☝️ . Since this is a new value we don't need to support folks passing it as a string. That was a holdover from how we did some old values so we needed that for backwards compat.

Reply:
Done!